spinta/backends/postgresql/types/geometry/encode.py

```diff
@@ -42,6 +44,14 @@ def prepare_dtype_for_response(
 ):
     shape = to_shape(value)
     point = shape.centroid
-    params = urlencode({'mlat': point.x, 'mlon': point.y})
-    link = f'https://www.openstreetmap.org/?{params}#map=19/{point.x}/{point.y}'
-    return Cell(shape.wkt, link=link)
+
+    if WKBElement.srid in [4326, -1]:
```

Comment:
Do not hardcode cryptic values, instead assign these numbers to a constant with self explanatory names.